packagesFromOverlaysBuilderConstructor.nix

```diff
@@ -1,89 +1,76 @@
 { flake-utils-plus }:
 let
 
-  overlayFromPackagesBuilderConstructor = channels:
+  overlayFromPackagesBuilderConstructor = overlays:
```

Comment:
If you don't mind, would you cover up my misnomer? It should be `packagesFromOverlaysBuilderConstructor`. I probably was convincingly conaitent woth that error down the file.

Sry!

:wink:

Reply:
fixed to follow the same naming